### PR TITLE
GH-1298: TerraWorkspaceSink - verify target entity columns exist

### PR DESCRIPTION
**Pull request by tbl3rd**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-1298
- 
### Changes

- Added another check to wfl.covid/verify-terra-sink! and fixed the tests it broke.

### Review Instructions

- The system tests fail with a database error now. Will investigate on Monday.
- Please let me know whether I'm on the right track.

### CODE REVIEW

File: api/src/wfl/module/covid.clj [hunk 1]
```diff
@@ -958,17 +959,23 @@
...
+          entity-types (firecloud/list-entity-types workspace)
+          types        (-> entity-types keys set)]
+      (when-not (types entity-type)
+        (throw (UserException. "Entity not found"
+                               (util/make-map entityType types workspace))))
+      (let [attributes    (get-in entity-types [entity-type :attributeNames])
+            [missing _ _] (data/diff (set (vals fromOutputs)) (set attributes))]
+        (when (seq missing)
+          (throw (UserException. "Attributes missing for these outputs"
+                                 (util/make-map missing)))))))
```
Comment (ehigham) [inline api/src/wfl/module/covid.clj:977]: Might be useful to know what the actual attributes are too, not just the missing ones. Even better sorted lexicographically.

**Review by ehigham — APPROVED** [review]
Thank you. Looks good.

**Review by rexwangcc — APPROVED** [review]
LGTM

File: api/src/wfl/module/covid.clj (hunk 1 above)
Comment (tbl3rd) [reply api/src/wfl/module/covid.clj]: Typically there will be lots of attributes but just a few missing ones,
but added and sorted.  We can revisit if `:missing` gets swamped.

**Pull request merged**